ogre_mesh_viewer.py

```diff
@@ -498,6 +505,8 @@ def mousePressed(self, evt):
                 return True
             
             new_entity = hit.movable.castEntity()
+            if not new_entity:
```

Comment:
unrelated change?

Reply:
No, this is to avoid crashing when the user accidentaly clicks the grid that is returned as a movable object but not entity

Reply:
please add a comment, as it is not obvious